### PR TITLE
NOISSUE - Enrich Existing OpenTelemetry Tags

### LINKED ISSUE #787: Enrich existing open tracing spans with additional data
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.
No.

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
After adding base open tracing support, spans should be enriched with additional tags. There is a recommendation of what should be added to the span and you can check it out [here](https://github.com/opentracing/specification/blob/master/semantic_conventions.md). One of the problems that we'll have to solve is how to write some kind of interceptor to the existing DB client in order to fetch SQL statements and passed params. The end goal should be removing separate logging and metrics middleware.

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).

### PR DESCRIPTION
**Pull request by rodneyosodo**
Signed-off-by: rodneyosodo <b1ackd0t@protonmail.com>

### What does this do?
Adds 
- `db.type`, `db.instance`, `db.user`,, `db.statement`, `peer.address`, `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service` and `span.kind` to all database traces
	- We used the database config to extract the hostname and port
- `message_bus.destination`, `span.kind`, `peer.address`, `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service` to message broker traces
	- we used the server config to extract the hostname and port
More details about the tags can be found https://github.com/open-telemetry/semantic-conventions/tree/main/docs
 
### Which issue(s) does this PR fix/relate to?
Resolves #787 

### List any changes that modify/break current functionality
- On `main` files, I have reordered the loading of environment variables to be after `logger` is initialized

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes

To be merged after https://github.com/mainflux/mainflux/pull/1841

Example of HTTP Publish

image

Things Repository Call

image

RabbitMQ Messaging System

image

### CODE REVIEW

File: cmd/cassandra-reader/main.go
Comment (SammyOina) [inline cmd/cassandra-reader/main.go:1]: are the changes on this required

File: cmd/certs/main.go
Comment (SammyOina) [inline cmd/certs/main.go:1]: same here

**Review by SammyOina — COMMENTED** [review]
Tested manually, and it works fine, LGTM. Just one more thing don't forget to add the env vars on README.MD

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**